tests/Acceptance/Extra/Workflow/AllHandlersFinishedTest.php

```diff
@@ -128,17 +131,88 @@ public function signalHandlersWithManyCalls(
             'Workflow result contains resolved values',
         );
     }
+
+    #[Test]
+    public function warnUnfinishedSignals(
+        #[Stub('Extra_Workflow_AllHandlersFinished')] WorkflowStubInterface $stub,
+    ): void {
+        $this->markTestSkipped("Can't check the log yet");
+
+        /** @see TestWorkflow::resolveFromSignal() */
+        $stub->signal('resolve', 'foo', 42);
+        $stub->signal('resolve', 'bar', 42);
+
+        for ($i = 0; $i < 8; $i++) {
+            /** @see TestWorkflow::addFromSignal() */
+            $stub->signal('await', "key-$i");
+        }
+
+        // Finish the workflow
+        $stub->signal('exit');
+        $stub->getResult(timeout: 1);
+
+        // todo Check that `await` signal with count was mentioned in the logs
```

Comment:
Looks like a bit of leftover? Below as well

Reply:
We currently don't have the capability to test this, so these TODOs will likely remain for the time being